src/components/CartCard/CartCard.css

```diff
@@ -0,0 +1,85 @@
+.cart-outer-container {
+  max-width: 35rem;
+  max-height: 17rem;
+  padding: 1rem;
+  border: 1px solid var(--light-grey);
+  box-shadow: rgba(0, 0, 0, 0.1) 0px 1px 2px 0px;
+}
+.cart-inner-container {
+  display: grid;
+  grid-template-columns: 45% 55%;
+  grid-template-areas: "image info";
+  gap: 0.3rem;
+}
+.cart-image-container {
+  grid-area: image;
+  box-shadow: rgba(0, 0, 0, 0.1) 0px 1px 2px 0px;
+  border: 1px solid var(--light-grey);
+}
+.cart-img {
+  object-fit: contain;
+  height: 100%;
+  object-fit: fill;
+  max-inline-size: 100%;
+  block-size: auto;
+  aspect-ratio: 1;
+}
+.cart-info {
+  grid-area: info;
+  margin: 0.7rem;
+}
+.cart-card-title {
+  font-size: 1.2rem;
+}
+
+.cart-remove-item,
+.cart-wishlist-btn {
+  display: inline-block;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 0.3rem;
+  outline: none;
+  border: none;
+  gap: 0.4rem;
+  width: 100%;
+  padding: 0.5rem;
+  font-size: 1.1rem;
+  margin: 0.4rem 0;
```

Comment:
if value is less than 10px you can use px unit just a suggestion you can skip this